### ISSUE TITLE
Two-Tower 추천 모델 실험 설계

### ISSUE THREAD

**Issue by haeinChoe** [haeinChoe/Travel-Recommendation#7]
## 목적
TravelMate 데이터 특성에 맞는 Two-Tower retrieval 모델의 입력, 학습, 평가 방식을 정의한다.

## User Tower 후보 feature
- age / gender
- travel style
- companions
- region / trip context
- interaction history

## Item Tower 후보 feature
- region
- category / theme
- TourAPI metadata
- text embedding

## 실험 항목
- embedding dimension
- negative sampling 방식
- loss function
- batch size / learning rate
- cold-start 처리

## 완료 조건
- 입력 feature와 label 정의가 확정되어 있다.
- baseline과 동일한 evaluation protocol로 비교 가능하다.
- online serving 시 필요한 artifact와 contract가 정의되어 있다.
